The `android_mblen()` implementation copied from the `imlib2` Termux package

--- a/src/ypaint.cc
+++ b/src/ypaint.cc
@@ -18,6 +18,11 @@
 
 #ifdef CONFIG_I18N
 #include <wctype.h>
+
+static int android_mblen(const char *s, size_t n)
+{
+    return mbtowc(0, s, n);
+}
 #endif
 
 static inline Display* display()  { return xapp->display(); }
@@ -310,7 +315,7 @@ void Graphics::drawCharUnderline(int x, int y, const char *str, int charPos) {
     int c = 0, cp = 0;
 
 #ifdef CONFIG_I18N
-    if (multiByte) mblen(nullptr, 0);
+    if (multiByte) android_mblen(nullptr, 0);
 #endif
     while (c <= len && cp <= charPos + 1) {
         if (charPos == cp) {
@@ -325,7 +330,7 @@ void Graphics::drawCharUnderline(int x, int y, const char *str, int charPos) {
             break;
 #ifdef CONFIG_I18N
         if (multiByte) {
-            int nc = mblen(str + c, size_t(len - c));
+            int nc = android_mblen(str + c, size_t(len - c));
             if (nc < 1) { // bad things
                 c++;
                 cp++;
